drivers/firmware/raspberrypi.c

```diff
@@ -232,8 +236,18 @@ static int rpi_firmware_get_throttled(struct rpi_firmware *fw, u32 *value)
 
 	ret = rpi_firmware_property(fw, RPI_FIRMWARE_GET_THROTTLED,
 				    value, sizeof(*value));
+
 	if (ret)
+	{
+		/* If the mailbox call fails once, then it will continue to
```

Comment:
Is having old firmware the only reason why this call shall fail? In which case I'm ok. 
Or are there other error returns apart from 0x80000001 that could be returned and should be considered?

Reply:
There is no nuance to the error returns, it's just a case of "Computer says no".